man/resolved.conf.xml

```diff
@@ -269,6 +269,22 @@
         in use.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DNSStubListenerExtra=</varname></term>
+        <listitem><para>Takes an IPv4 or IPv6 address to listen on. The address may optionally be prefixed by <literal>:</literal> and
+        a protocol name (<literal>udp</literal> or <literal>tcp</literal>). It may also be optionally suffixed by <literal>:</literal>
```

Comment:
`may be optionally prefixed by one of the protocol name <literal>udp</literal> or <literal>tcp</literal> with separator <literal>:</literal>.`